ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -190,6 +200,143 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.ability({ id: '788B', source: 'Hegemone', capture: false }),
       response: Responses.moveAway(),
     },
+    {
+      id: 'P6S Predation Debuff Collect',
+      // CF7 Glossal Resistance Down (Snake Icon)
+      // CF8 Chelic Resistance Down (Wing Icon)
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['CF7', 'CF8'] }),
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => data.predationDebuff = matches.effectId,
+    },
+    {
+      id: 'P6S Predation Bait Order',
+      // Using Aetheronecrosis (CF9)
+      // These come out as 20s, 16s, 12s, or 8s
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'CF9' }),
+      condition: Conditions.targetIsYou(),
+      preRun: (data, matches) => data.aetheronecrosisDuration = parseFloat(matches.duration),
+      delaySeconds: 0.1,
+      durationSeconds: (_data, matches) => {
+        const duration = parseFloat(matches.duration);
+        // First Dual Predation is 3.7s before expiration
+        // Remaining Dual Predations are 12.3s (second), 12.4s (third/fourth)
+        return duration === 20 ? duration - 3.8 : duration + 12.3;
+      },
+      infoText: (data, matches, output) => {
+        const AetheronecrosisMap: { [duration: number]: string } = {
+          20.00: output.firstBait!(),
+          8.00: output.secondBait!(),
+          12.00: output.thirdBait!(),
+          16.00: output.fourthBait!(),
+        };
+        const dir = data.predationDebuff === 'CF7' ? output.left!() : output.right!();
+        return output.text!({ dir: dir, bait: AetheronecrosisMap[parseFloat(matches.duration)] });
+      },
+      outputStrings: {
+        text: {
+          en: '${dir}, ${bait}',
+        },
+        left: {
+          en: 'Left (Wing Side)',
```

Comment:
Is the boss always facing North here? Should these be West/East instead?

Reply:
The boss always faces North, so whether relative to boss facing or true north, both will work. I just tend to think boss side to look at.

Reply:
If you want to follow up and rename, go for it.